Makefile.am

```diff
@@ -412,6 +415,7 @@ tools_autotools_tsk_gettimes_SOURCES = tools/autotools/tsk_gettimes.cpp
 tools_autotools_tsk_imageinfo_LDADD = $(TSK_LIB)
 tools_autotools_tsk_imageinfo_SOURCES = tools/autotools/tsk_imageinfo.cpp
 
+tools_autotools_tsk_loaddb_CXXFLAGS = $(AM_CXXFLAGS)
```

Comment:
This line is unnecessary, since `$AM_CXXFLAGS` is the default value.

Reply:
fixed in 2584e0e67.